src/test/resources/com/atomist/rug/kind/core/Replacer.ts

```diff
@@ -1,10 +1,11 @@
-import {ProjectEditor} from '@atomist/rug/operations/ProjectEditor'
+import {Editor} from '@atomist/rug/operations/Decorators'
 import {Project} from '@atomist/rug/model/Core'
 import {Parameter} from '@atomist/rug/operations/RugOperation'
 import {PathExpressionEngine} from '@atomist/rug/tree/PathExpression'
 import {Replacer} from '@atomist/rug/model/Core'
 
-class Replacer1 implements ProjectEditor {
+@Editor("Replacer")
+class Replacer1  {
```

Comment:
Well, it's useful when trying to ensure we get the implementation right, but due to the type erasure, we can't enforce it - at least not with our current registration method.